pages/common/salt-call.md

```diff
@@ -1,6 +1,7 @@
 # salt-key
```

Comment:
I realize this is not in the scope of this PR, but please fix the command name to match the page (file) name. You could do that as a separate commit.

As for the other commits, I'm not sure it's justified to have one commit per page, since they're all doing the same thing. What do you think about squashing them into one?

Reply:
Do you object to combining the link-adding commits?

Reply:
Combining all of the commits? Or just the `salt` ones? 🙂

Reply:
I was thinking all the ones that add links to the pages, so there would be one commit with the adding of links (with the list of pages in the extended commit message, like this one: ce642b6c1), and one commit with the other fixes (page title and whitespace). But that's my opinion (I think semantically they're one "single" change); if you disagree, let me know.